### PR TITLE
Add Phone masking + Bundle promo strings

### PR DESCRIPTION
**Pull request by codemist**
Figma specs:
image
image

### CODE REVIEW

File: en/messages.json [hunk 1]
```diff
@@ -298,6 +298,37 @@
...
     },
+    "popupBundlePromoHeadline": {
+        "message": "Add Mozilla VPN at $SAVINGS$ off.",
+        "description": "DO NOT TRANSLATE \"Mozilla VPN\"",
+        "placeholders": {
+            "savings": {
+                "content": "50%"
+            }
+        }
+    },
```
Comment (codemist) [inline en/messages.json:309]: Is this the right way to call out variables?

File: en/messages.json [hunk 2]
```diff
@@ -298,6 +298,37 @@
...
+        "message": "Introducing phone number masking"
+    },
+    "popupPhoneMaskingPromoBody": {
+        "message": "With phone number masking, you can now create a phone number mask that helps you get texts and calls without revealing your true number."
+    },
+    "popupPhoneMaskingPromoCTA": {
+        "message": "Upgrade now"
+    },
     "popupSignBackInHeadline": {
         "message": "Sign back in with your aliases"
```
Comment (peiying2) [inline en/messages.json:333]: should we update aliases to masks?

File: en/messages.json (hunk 1 above)
Comment (peiying2) [reply en/messages.json:309]: In checking how other variables are referenced in the file, it looks good!

**Review by peiying2 — APPROVED** [review]
The PR looks good to go.

I do want to raise the issue of old strings still present in the files, old terms like aliases. It's been many months since the term change. Let's make a plan to remove them. There are new requests for Relay and add-on and I hate to see the communities wasting time to translate them.

File: en/messages.json (hunk 2 above)
Comment (peiying2) [reply en/messages.json:333]: Thank you so much. 

Keep an eye on the following. 
- alias and aliases
- real email and real emails

**Comment by codemist** [comment]
@peiying2 noted. Will get to refreshing the older strings. Sorry, the add on took a backseat in favour of multiple Relay projects.

**Pull request merged**